README.md

```diff
@@ -15,70 +15,77 @@
-- [Examples](https://github.com/amethyst/laminar#examples)
-    - [Udp](https://github.com/amethyst/laminar#udp)
-- [Notice](https://github.com/amethyst/laminar#notice)
-- [Contributing](https://github.com/amethyst/laminar#contributing)
-- [Authors](https://github.com/amethyst/laminar/#authors)
+- [Useful links](#useful-links)
+- [Features](#features)
+- [Getting Started](#getting-stated)
+- [Examples](#examples)
+- [Notice](#notice)
+- [Contributing](#contribution)
+- [Authors](#authors)
 - [License](#license)
 
+## Features
+These are the features this crate provides:
+
+- UDP-based protocol
+- Automatic Fragmentation
+- RTT estimation
+- Connection tracking.
+- Unreliable and Reliable sending of packets.
+- Protocol version monitoring.
+- A link conditioner to simulate packet loss and latency
+- Good error handling with **zero** panics.
+- Well tested by integration tests and unit tests
+- Benchmarks
+
+## Getting Stated
 Add the laminar package to your `Cargo.toml` file.
 
 ```toml
 [dependencies]
-laminar = "0.0.0"
-```
-And import the laminar modules you want to use.
-
-```rust
-extern crate laminar;
-
-// this module contains all socket related logic.
-use laminar::net::{UdpSocket, SocketAddr, NetworkConfig, Connection, Quality};
-// this module contains packet related logic.
-use laminar::packet::{Packet};
+laminar = "0.1.0"
 ```
 
-## Useful Links
+### Useful Links
 
 - [Documentation](https://docs.rs/laminar/).
 - [Cargo Page](https://crates.io/crates/laminar)
 - [Examples](https://github.com/amethyst/laminar/tree/master/examples)
-
-## Features
-These are the features from this crate:
-
-- Semi-reliable UDP
-- Fragmentation
-- RTT estimation
-- Virtual connection management.
+- [Contributing Guides](https://github.com/amethyst/laminar/blob/master/docs/StyleGuides)
 
 ## Examples
-These are some basic examples demonstrating how to use this crate. See [examples](https://github.com/amethyst/laminar/tree/master/examples) for more.
+These are some basic examples demonstrating how to use this crate. 
+Please checkout our [examples](https://github.com/amethyst/laminar/tree/master/examples) for more.
 
-### Udp API | [see more](https://github.com/amethyst/laminar/blob/master/examples/udp.rs)
+### UDP API | [see more](https://github.com/amethyst/laminar/blob/master/examples/udp.rs)
 This is an example of how to use the UDP API.
 
 _Send packets_
 
 ```rust
+use laminar::{DeliveryMethod, Packet};
+use laminar::net::{UdpSocket, NetworkConfig};
+
 // Create the necessarily config, you can edit it or just use the default.
 let config = NetworkConfig::default();
 
 // Setup an udp socket and bind it to the client address.
 let mut udp_socket = UdpSocket::bind("127.0.0.1:12346", config).unwrap();
 
 // Create a packet that can be send with the given destination and raw data.
-let packet = Packet::new(destination, vec![1,2,3]);
+let packet = Packet::new(destination, vec![1,2,3], DeliveryMethod::Unreliable);
```

Comment:
I would move the vec out and do something like
```rust
let bytes = vec![...];
let packet = Packet::new(destination, bytes, DeliveryMethod::Unreliable);
```
as this shows the usage a bit clearer.